bindings/python/py_src/tokenizers/implementations/sentencepiece_unigram.py

```diff
@@ -84,6 +84,7 @@ def train_from_iterator(
         vocab_size: int = 8000,
         show_progress: bool = True,
         special_tokens: List[Union[str, AddedToken]] = [],
+        initial_alphabet: List[str] = [],
```

Comment:
Python should never have mutable default arguments: https://docs.python-guide.org/writing/gotchas/

Using `Optional` with a None default is the correct way to handle this I think.

Reply:
And... `special_tokens` should be modified to (just one line above).